docs/how-to/custom-actions.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Custom Actions Explained
+sidebar_label: Custom Actions Explained
+slug: /custom-actions
+---
+
+# Custom Actions Explained
+
+Custom Actions give you the ability to enable or disable specific default API functions related to CRUD operations. This feature significantly enhances your control over the generated REST and GraphQL APIs to ensure that your API contains only the necessary code tailored to your requirements.
+
+![Custom Actions](./assets/custom-actions.png)
+
+:::note
+The Custom Actions feature is available in Amplication's **Enterprise** plan.
+:::
+
+## How To Enable Custom Actions For An Entity
+
+To utilize Custom Actions for an entity, begin by accessing the new **Modules** tab in your Amplication dashboard. Here, each entity is associated with a _Module_, allowing you to perform _Module Actions_, operations on these entities with or without API endpoints.
```

Comment:
Not sure I understand the "with or without API endpoints". I would omit it.
@morhag90 - WDYT?

Reply:
The ',' after the "_Module Actions_" should be removed